squad/api/rest.py

```diff
@@ -833,6 +833,23 @@ class AnnotationViewSet(viewsets.ModelViewSet):
     ordering_fields = ('id', 'build')
 
 
+class MetricThresholdSerializer(serializers.ModelSerializer):
```

Comment:
Not a requirement but maybe using HyperlinkedModelSerializer is better? It's surely more consistent with the other serializers we have. If there is a good reason not to use it, I'm OK with that. If you decide to move to hyperlinked one, please add 'id' field manually.